src/plugins/intel_npu/tools/single-image-test/main.cpp

```diff
@@ -156,6 +166,25 @@ std::vector<std::string> splitStringList(const std::string& str, char delim) {
     return out;
 }
 
+std::map<std::string, std::string> parseArgMap(std::string argMap) {
+    argMap.erase(std::remove_if(argMap.begin(), argMap.end(), ::isspace), argMap.end());
+
+    const auto pairs = splitStringList(argMap, ';');
```

Comment:
![image](https://github.com/user-attachments/assets/338a600b-3eab-4180-9a73-5b393ae19358)
Just a juggestion, I guess we can consider comma as separator symbol the same way other tools does.

Reply:
--img_bin_precision also has comma as separator for example.

Reply:
thanks for the comment, I think I didn't use a comma here because layout naming values may contain a comma already, like `[N,HEIGHT,WIDTH]` as part of an advanced layout syntax in OV: https://github.com/openvinotoolkit/openvino/blob/master/src/core/src/layout.cpp#L103